### PR TITLE
Fix tests in IE11

### PR DESCRIPTION
**Pull request by john-griffin**
Gets the tests passing in IE11. Based on feedback in #1391.

### CODE REVIEW

File: tests/integration/orm/mixed/2-many-to-one/accessor-test.js [hunk 1]
```diff
@@ -14,7 +14,6 @@ module('Integration | ORM | Mixed | Many To One | accessor', function(hooks) {
     test(`the references of a ${state} are correct`, function(assert) {
       let [ post, user ] = this.helper[state]();
 
-      assert.deepEqual(post.user, user);
```
Comment (john-griffin) [inline tests/integration/orm/mixed/2-many-to-one/accessor-test.js]: A regular `equal` call had issues here too. Do you have any suggestions?

Comment (samselikoff) [reply tests/integration/orm/mixed/2-many-to-one/accessor-test.js]: Hm try `assert.ok(post.user.equals(user))`

**Comment by samselikoff** [comment]
nice work man!

File: tests/integration/orm/mixed/2-many-to-one/accessor-test.js (hunk 1 above)
Comment (samselikoff) [reply tests/integration/orm/mixed/2-many-to-one/accessor-test.js]: Ah I see. I might do something like this:
```
if (post.user) {
  assert.ok(post.user.equals(user));
} else {
  assert.equal(post.user, null);
  assert.equal(user, null);
}
```

**Comment by john-griffin** [comment]
Thanks, that last change is in, can confirm all passing in IE11 now.

**Comment by samselikoff** [comment]
Nice work, and thanks!

**Pull request merged**